externs/svg-externs.js

```diff
@@ -0,0 +1,10367 @@
+/**
+ * @fileoverview Generated by closureidl 0.2dev closureidl-parser-webkit 0.2dev dart-idl 7592.
```

Comment:
This extern file contains the SVG API calls, so Closure Compiler with advanced optimizations will not mangle the standard SVG function names. To use advanced optimizations it is necessary to have it (from any source).

- Generating it during the build is IMHO just overcomplicating Blockly's build process and adding dependencies that are not really useful (closureidl, svg sources etc).
- It could be better to discuss this with the Closure Compiler's team, that they would include SVG externs in the compiler's default externs